nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/resources/elasticsearch-test-nested-contrib.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.elasticsearch.test.override.hints.contrib">
```

Comment:
Same remark as https://github.com/nuxeo/nuxeo/pull/3481#discussion_r336395556 to be consistent.